### ISSUE TITLE
[Feature Request] Save settings online/cloud

### ISSUE THREAD

**Issue by shawnboy99** [rNeomy/auto-tab-discard#102]
Would love a feature to save settings Firefox account/cloud/online to be reloaded on new Firefox installs.

**Comment by fermulator** [comment]
And specifically part of Firefox-Sync https://support.mozilla.org/en-US/kb/what-firefox-sync

**Comment by rugk** [comment]
Indeed, that is actually also trivial to implement, as the WebExtension API (and Firefox) have 100% support for this: See https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/sync

**Comment by soredake** [comment]
@rNeomy is there progress on this?

**Comment by rNeomy** [comment]
@soredake not yet! It is pretty simple but converting to the sync storage resets all the configs back to the default values. So I am still investigating

**Comment by soredake** [comment]
Can this be added, please?

**Comment by joaovl5** [comment]
bump
